features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+@owner:lilla
+@author:lilla
+
+Feature: Navigating Sidenav
+    This allows users to easily navigate and select between articles and categories within our help center site
+
+    @ui
+    @motivating
+    Scenario Outline: Default menu item state in the side navigation bar
```

Comment:
It would be best to quickly describe the default here. Remember scenario titles should stand alone as a quick description of the scenario/spec (applies to other scenario titles in this file too)